doc/classes/EditorSettings.xml

```diff
@@ -1050,6 +1050,9 @@
 			If [code]true[/code], redraws the editor every frame even if nothing has changed on screen. When this setting is enabled, the update spinner displays in red (see [member interface/editor/show_update_spinner]).
 			[b]Warning:[/b] This greatly increases CPU and GPU utilization, leading to increased power usage. This should only be enabled for troubleshooting purposes.
 		</member>
+		<member name="interface/editor/use_alt_shortcuts" type="int" setter="" getter="">
```

Comment:
I would give the editor setting a more explicit name that's more likely to show up in search results:

```suggestion
		<member name="interface/editor/show_shortcuts_as_symbols" type="int" setter="" getter="">
```